services/spark-historyserver.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "spark-historyserver",
+  "description": "Spark History Server",
+  "dependencies": {
+    "conflicts": [],
+    "install": {
+      "requires": [ "base" ],
+      "uses": [ "kerberos-client" ]
+    },
+    "provides": [],
+    "runtime": {
+      "requires": [],
+      "uses": [ "kerberos-master" ]
+    }
+  },
+  "provisioner": {
+    "actions": {
+      "install": {
+        "type":"chef-solo",
+        "fields": {
+          "run_list": "recipe[hadoop::spark_historyserver]"
+        }
+      },
+      "configure": {
+        "type": "chef-solo",
+        "fields": {
+          "run_list": "recipe[hadoop_wrapper::default],recipe[hadoop::spark]"
```

Comment:
Nope. Config is done in the `spark` recipe, which `spark_historyserver` includes. To just re-run the configuration, we can call just `spark` and avoid processing a chunk of resources that should be no-op, anyway.